pkg/ruler/api.go

```diff
@@ -135,11 +135,25 @@ func (a *API) PrometheusRules(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	state := strings.TrimSpace(strings.ToLower(req.URL.Query().Get("state")))
```

Comment:
Do we need `strings.TrimSpace`? We don't do it for other parameters.

Reply:
Yep, removing since other params doesn't do it.